Manipulator/KernelManipulator.php

```diff
@@ -23,11 +23,6 @@ class KernelManipulator extends Manipulator
     protected $kernel;
     protected $reflected;
 
-    /**
-     * Constructor.
-     *
-     * @param KernelInterface $kernel A KernelInterface instance
```

Comment:
makes no senso to me, but I'm fine with this.

I was just interested in this, thanks for the feedback